service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -209,6 +216,31 @@ private void sendCMOCaseLinkNotifications(final EventData eventData) {
         sendCMOCaseLinkNotificationToRepresentatives(eventData);
     }
 
+    @EventListener
+    public void sendNotificationToPartiesAddedToCase(PartyAddedToCaseEvent event) {
+        List<Element<Representative>> representatives = event.getRepresentativesToNotify();
+        EventData eventData = new EventData(event);
+
+        if (isNotEmpty(representatives)) {
```

Comment:
if the empty list is returned as mentioned else where this if check isn't needed